File: the-lambda-trilogy/typescript/lambdas/single-purpose-function/add.ts
export {};

exports.handler = async (event:any) => {
    console.log(JSON.stringify(event, null, 2));

    return sendRes(200, "4");
}

const sendRes = (status:number, body:string) => {
    var response = {
      statusCode: status,
      headers: {
        "Content-Type": "text/html"
      },
      body: body
    };
    return response;
  };

File: the-lambda-trilogy/python/lambda_fns/the_single_purpose_function/add.py
def handler(event, context):
    try:
        first_num = event["queryStringParameters"]['firstNum']
    except KeyError:
        first_num = 0

    try:
        second_num = event["queryStringParameters"]['secondNum']
    except KeyError:
        second_num = 0

    result = int(first_num) + int(second_num)
    print("The result of % s + % s = %s" % (first_num, second_num, result))
    return {'body': result, 'statusCode': 200}